src/conn/conn_api.c

```diff
@@ -3010,8 +3010,8 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler, const char *c
         WT_ERR(wt_session->salvage(wt_session, WT_METAFILE_URI, NULL));
     }
 
-    /* Initialize the connection's base write generation. */
-    WT_ERR(__wt_metadata_init_base_write_gen(session));
+    /* Initialize connection values from stored metadata */
```

Comment:
Missing a period at the end of the sentence.

Reply:
Fixed in [a30614b](https://github.com/wiredtiger/wiredtiger/pull/8245/commits/a30614b0b9f9c9da6a61ed0e7fb4575c41456224)